### PR TITLE
Update link in _index.md

### PR DESCRIPTION
**Pull request by snowei**
Change link from English version to Chinese version

### CODE REVIEW

**Comment by snowei** [comment]
Signed CLA

File: content/zh/docs/concepts/_index.md [hunk 1]
```diff
@@ -72,10 +72,10 @@ The basic Kubernetes objects include:
 
 基本的 Kubernetes 对象包括：
 
-* [Pod](/docs/concepts/workloads/pods/pod-overview/)
-* [Service](/docs/concepts/services-networking/service/)
-* [Volume](/docs/concepts/storage/volumes/)
```
Comment (tanjunchen) [inline content/zh/docs/concepts/_index.md:77]: we  need to update this?
it  seems not good.
/cc @tengqm

Comment (tanjunchen) [reply content/zh/docs/concepts/_index.md:77]: @snowei
Thanks for your contribution

Comment (tengqm) [reply content/zh/docs/concepts/_index.md:77]: We still have to do it today.

Comment (tanjunchen) [reply content/zh/docs/concepts/_index.md:77]: done . get  it .

**Comment by markthink** [comment]
@snowei 参考[此文档](https://github.com/k8smeetup/k8s-official-translation)

**Comment by kbhawkey** [comment]
Hi @snowei . Would you verify that you have a valid CLA? Thanks!

**Comment by tengqm** [comment]
Closing this due to CLA status failure for more than one month. Please feel free to reopen it when the CLA issue is fixed.
/close

**Pull request closed without merging**